src/azure-cli-core/azure/cli/core/tests/test_generic_update.py

```diff
@@ -63,6 +63,11 @@ def __init__(self):
         self.empty_dict_of_dicts = {'dict': {'dict2': None}}
         self.empty_dict = {'dict3': None}
 
+        self.additional_properties = {'additional1': 'addition value #1'}
+
+    def enable_additional_properties_sending(self):
+        pass
```

Comment:
Is this intentionally a no-op?

Reply:
Okay, let me just make our test object be an instance of SDK's Model type. This not only gets rid of this confusion, but also mirrors the most common update scenarios